### DISCUSSION TITLE
Eating when god is enabled

### DISCUSSION THREAD

**Discussion by thunderlight411** [Zrips/CMI#2593] [Enhancement]
When a player have godmode enabled there can eat any type of food.
when they turn off the godmode it is now longer possible when the hunger bar is full.

---

**Cmi Version (using`/cmi version`): ***8.6.16.4***

**Server Type (Spigot/Paperspigot/etc): ***Paperspigot***

**Server Version (using `/ver`): ***git-paper-26***

**Comment by Zrips** [comment]
In general this is not a bug but vanilla behavior, or more like a vanilla bug.

**Comment by thunderlight411** [comment]
That is strange on a full vanilla server server without any plugins and with the same server version this problem does not exist.
